src/main/java/de/retest/recheck/printer/ElementDifferencePrinter.java

```diff
@@ -24,6 +28,17 @@ private String createDescription( final ElementDifference difference ) {
 	}
 
 	private String createDifferences( final ElementDifference difference, final String indent ) {
+		final LeafDifference identifyingAttributesDifference = difference.getIdentifyingAttributesDifference();
+		if ( identifyingAttributesDifference instanceof InsertedDeletedElementDifference ) {
+			final Printer<InsertedDeletedElementDifference> printer = new InsertedDeletedElementDifferencePrinter();
+			return printer.toString( (InsertedDeletedElementDifference) identifyingAttributesDifference, indent );
+		}
+
+		final List<AttributeDifference> differences = difference.getAttributeDifferences( Filter.FILTER_NOTHING );
+		if ( differences.isEmpty() ) {
+			return indent + "no differences";
```

Comment:
Does this make sense?

Reply:
That should now be fixed with the other PRs.